tests/test_interactions.py

```diff
@@ -340,3 +341,120 @@ def test_cyin007_click_twice(dash_duo):
 
     assert clicked_label_1 == clicked_label_2
     assert clicked_timestamp_1 != clicked_timestamp_2
+
+
+def test_cyin008_ctx_menu_remove_node(dash_duo):
+    init_pos, actions, _ = create_app(dash_duo)
+    init_x, init_y = init_pos["Node 1"]
+
+    # Open the Drag data JSON tab
+    actions.move_to_element(dash_duo.find_element("#tabs > div:nth-child(5)"))
+    actions.click().perform()
+    time.sleep(1)
+
+    # Select the JSON output element before removal
+    elements = dash_duo.find_element("pre#elements-data-json-output")
+    nb_elements_before = len(json.loads(elements.text))
+
+    # move mouse to first node and right click and click on remove
+    actions.move_to_location(init_x, init_y)
+    actions.context_click()
+    actions.move_to_element(dash_duo.find_element("button#remove"))
+    actions.click()
+    actions.perform()
+    time.sleep(1)
+
+    # Select the JSON output element after removal
+    elements = dash_duo.find_element("pre#elements-data-json-output")
+    nb_elements_after = len(json.loads(elements.text))
+
+    assert nb_elements_before > nb_elements_after
```

Comment:
That won't work because when you delete a node, it also deleted all its outgoing/incoming edges.

Reply:
Gotcha -- in that case then calculate how many edges are deleted and use that number. Better to check for an exact number rather than just greater / less than.